src/reward.cpp

```diff
@@ -0,0 +1,60 @@
+#include "otpch.h"
+
+#include <iostream>
+#include "reward.h"
+
+
+Reward::Reward() :
+	Container(ITEM_REWARD_CONTAINER)
+{
+	maxSize = 32;
+	unlocked = true;
```

Comment:
Set to false to have a different animation when a player tries to put something inside the bag.